[GLIB] Update list of dnf dependencies

TingPing opened the pull request:
#### fa2a14f12031baf24d5bc6d6727c3e25c5052589
<pre>
[GLIB] Update list of dnf dependencies
<a href="https://bugs.webkit.org/show_bug.cgi?id=247689">https://bugs.webkit.org/show_bug.cgi?id=247689</a>

Reviewed by Michael Catanzaro.

This updates the list of dependencies for WPE/GTK reducing duplication
and adding missing dependencies.

Tested on Fedora 37 toolbox.

* Tools/glib/dependencies/dnf:
* Tools/gtk/dependencies/dnf:
* Tools/wpe/dependencies/dnf:

Canonical link: <a href="https://commits.webkit.org/256536@main">https://commits.webkit.org/256536@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/591297dcfe6df97a66997af27ee79923245dfd18

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/95974 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/5223 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/29016 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/105535 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/165858 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/99953 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/5331 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/33985 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/88356 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/101358 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/101635 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/3931 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/82580 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/30977 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/85780 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/87689 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/73810 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/39716 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/19216 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/37395 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/20558 "Passed tests") | | 
| | [💥 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/41992 "An unexpected error occured. Recent messages:Failed to print configuration") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/43172 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/2173 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/43863 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/39818 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

mcatanzaro reviewed (approved):
Isn't the jhbuild only used by the Ubuntu and Debian LTS bots? It's probably time to delete it and find some other mechanism for managing these bots...?

TingPing committed:
[GLIB] Update list of dnf dependencies
https://bugs.webkit.org/show_bug.cgi?id=247689

Reviewed by Michael Catanzaro.

This updates the list of dependencies for WPE/GTK reducing duplication
and adding missing dependencies.

Tested on Fedora 37 toolbox.

* Tools/glib/dependencies/dnf:
* Tools/gtk/dependencies/dnf:
* Tools/wpe/dependencies/dnf:

Canonical link: https://commits.webkit.org/256536@main
